tools/ci_build/github/azure-pipelines/win-ci-fuzz-testing.yml

```diff
@@ -0,0 +1,157 @@
+jobs:
+- job: 'build'
+  pool: 'Win-CPU-2019'
+  strategy:
+    maxParallel: 2
+    matrix:
+      debug:
+        BuildConfig: 'Debug'
+      release:
+        BuildConfig: 'RelWithDebInfo'
+  variables:
+    OrtPackageId: 'Microsoft.ML.OnnxRuntime'
+    MsbuildArguments: '-detailedsummary -maxcpucount -consoleloggerparameters:PerformanceSummary'
+    OnnxRuntimeBuildDirectory: '$(Build.BinariesDirectory)'
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+    EnvSetupScript: setup_env.bat
+    buildArch: x64
+    setVcvars: true
+    ALLOW_RELEASED_ONNX_OPSET_ONLY: '0'
+  timeoutInMinutes: 2160
```

Comment:
>2160 [](start = 20, length = 4)

Is this a bit too long for timeout?

Reply:
Earlier the build pipeline timed-out. I wanted find out how long it will run. I set a higher limit. I was not meant to checkin.


---
In reply to: [524839479](https://github.com/microsoft/onnxruntime/pull/5771#discussion_r524839479) [](ancestors = 524839479)